ci/run.sh

```diff
@@ -58,10 +58,15 @@ clone() {
 
 # build dmd (incl. building and running the unittests), druntime, phobos
 build() {
+    BUILD_DFLAGS=
     if [ "$OS_NAME" != "windows" ]; then
         source ~/dlang/*/activate # activate host compiler, incl. setting `DMD`
     fi
-    $DMD compiler/src/build.d -ofgenerated/build
+    if [ "$OS_NAME" == "osx" ]; then
+	BUILD_DFLAGS="-L-platform_version -Lmacos -L${MACOSX_DEPLOYMENT_TARGET+10.9} -L0.0"
```

Comment:
```suggestion
	BUILD_DFLAGS="-L-platform_version -Lmacos -L${MACOSX_DEPLOYMENT_TARGET+11} -L0.0"
```
On phone so making changes via pr review